src/server/views/help/terms-and-conditions.njk

```diff
@@ -0,0 +1,153 @@
+{% extends 'layout.njk' %}
```

Comment:
As above: content needs to be aligned with the design: https://docs.google.com/document/d/1qzzR5-oyN6RAmE6uwlQznXgLzrO-YXu_YkRvwtjRZVs/edit#heading=h.u7j50dlb9t0v